qa/ceph_salt/salt.yaml

```diff
@@ -2,5 +2,5 @@ tasks:
 - clock:
 - install:
     install_ceph_packages: false
-    extra_system_packages: [salt, salt-master, salt-minion, salt-api]
+    extra_system_packages: [salt, salt-master, salt-minion, salt-api, which, lsof]
```

Comment:
what is `which` needed for? I tend to avoid it like the plague.

Reply:
iirc it's is needed for ceph-salt and it was missing from the image. I will test if it works without it and remove if if it's not needed anymore.

Reply:
not needed for ceph-salt since https://github.com/ceph/ceph-salt/pull/85 was merged